### PR TITLE
Added abilities to copy transaction id by click the transaction id

### PR DESCRIPTION
**Pull request by josikie**
Hello! This pr added abilities to copy transaction id to clipboard by click it.

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -31,29 +31,31 @@ android {
         }
     }
     assetPacks = [":install_time_asset_pack",":fast_follow_asset_pack_01"]
-    signingConfigs {
```
Comment (kcw-grunt) [inline app/build.gradle:34]: Hey @josikie .  Will you revert?  Per the notes in our chat to allow you to compile

File: app/build.gradle [hunk 2]
```diff
@@ -31,29 +31,31 @@ android {
...
-        if (props != null && props.containsKey('storeFile') && props.containsKey('storePassword') &&
-                props.containsKey('keyAlias') && props.containsKey('keyPassword')) {
-
-            android.signingConfigs.release.storeFile = file(props['storeFile'])
-            android.signingConfigs.release.storePassword = props['storePassword']
-            android.signingConfigs.release.keyAlias = props['keyAlias']
-            android.signingConfigs.release.keyPassword = props['keyPassword']
-        }
-    }
+    // remove the comment from line 36 - 58 and 222 for the signing release
```
Comment (kcw-grunt) [inline app/build.gradle:35]: Hey @josikie .  Will you revert?  Per the notes in our chat to allow you to compile

File: app/build.gradle [hunk 3]
```diff
@@ -217,7 +219,7 @@ android {
 
     buildTypes {
         release {
-            signingConfig signingConfigs.release
+            //signingConfig signingConfigs.release
```
Comment (kcw-grunt) [inline app/build.gradle:222]: Hey @josikie .  Will you revert?  Per the notes in our chat to allow you to compile

**Review by kcw-grunt — CHANGES_REQUESTED** [review]
Thanks @josikie . Will you take a look at the notes? Tx!

File: app/build.gradle (hunk 3 above)
Comment (josikie) [reply app/build.gradle:222]: Hello! 
Do you mean to delete them? @kcw-grunt 
I commented them so I could compile. Don't worry, I already ran it after put the changes

Comment (josikie) [reply app/build.gradle:222]: I'll uncomment

**Comment by josikie** [comment]
@kcw-grunt It's work smoothly, I already ran it without the signing release code in build.gradle

**Pull request merged**